gdx/src/com/badlogic/gdx/utils/DefaultPool.java

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
I thought some automated thing took care of license headers, but maybe not... I think copying the existing license header from any file that has it is fine, if Spotless isn't handling it.